Button/button.css

```diff
@@ -0,0 +1,180 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root{
+    --gray:#3e4152;
+    --text-color:#007FFF;
+    --gray-bg:#e5e5e5;
+    --gray-hover:#adb5bd;
+    --blue-bg:#023e8a;
+    --bg-code:#FAFAFA;
+}
+
+body{
+    font-family: 'Poppins', sans-serif;
+    margin: 2rem;
+    color:var(--gray);
+}
+h1{
+    font-size:3rem;
+}
+
+h1, h2, p{
+    margin: 0.625rem 0;
+    line-height: 1.2;
+}
+
+h2{
+    font-size: 2rem;
+}
+p{
+    font-size: 1.2rem;
+    padding: 0.8rem;
+}
+.btn-box{
+    border:1px solid var(--gray);
+    margin: auto;
+    width:80%;
+    padding: 0.2rem;
+    border-radius: 6px;
+    margin-bottom: 1.5rem;
+}
+.fas{
+    font-size: 1rem;
+    padding: 0.4rem;
+}
+
+button{
+    font-size: 1rem;
+    border: none;
+    outline: none;
+    background: white;
+    cursor: pointer;
+}
+
+.btn-variant{
+    margin: 1.5rem 0.8rem;
+    border-radius: 6px;
+}
+.btn-solid-primary, .btn-floating{
+    color:white;
+    background: var(--text-color);
+    border: white;
+    padding: 0.8rem;
+}
+.btn-outline-primary{
+    color: var(--text-color);
+    border:2px solid var(--text-color);
+    padding: 0.6rem;
+}
+.btn-outline-primary:hover{
+    background-color: var(--text-color);
+    color: white;
+}
+
+.link-text-primary{
+    color: var(--gray);
+    border-bottom:2px solid var(--text-color);
+    margin: 1.5rem 0.5rem;
+    font-size: 1.2rem;
+    padding: 0.3rem;
+}
+.link-text-primary:hover{
+    color: var(--text-color);
+}
+
+.btn-icon-primary i{
+    color: var(--text-color);
+    margin: 1.5rem 0.5rem;
+}
+.btn-text-icon-primary{
+    color: white;
+    background: var(--text-color);
+    padding: 0.6rem 0.8rem;
+}
+
+.btn-solid-secondary{
+    background:var(--gray-bg);
+    font-size: 1.1rem;
+    border: 1px solid var(--gray-bg);
+    padding: 0.8rem;
+}
+
+.btn-outline-secondary{
+    border:1px solid var(--gray);
+    padding: 0.6rem;
+}
+
+.btn-solid-secondary:hover, .btn-outline-secondary:hover{
+    border: 1px solid black;
+    background: var(--gray-hover);
+    color: black;
+}
+
+.link-text-secondary{
+    color: var(--gray);
+    border-bottom:2px solid var(--gray);
+    margin:0.8rem;
+    font-size: 1.2rem;
+    padding: 0.3rem;
+}
+
+.btn-icon-secondary i{
+    color: var(--gray);
+    margin:0.8rem;
+}
+.btn-text-icon-secondary{
+    color: var(--gray);
+    padding: 0.8rem;
+}
+.link-text-secondary:hover, .btn-icon-secondary i:hover, .btn-text-icon-secondary:hover{
+    color: black;
```

Comment:
Suggestion:
Multiple times 'white' and 'black' has been used. I think its better to declare a variable and use those.